var/spack/repos/builtin/packages/openjpeg/package.py

```diff
@@ -44,9 +44,3 @@ class Openjpeg(Package):
     version('1.5.1', 'd774e4b5a0db5f0f171c4fc0aabfa14e')
 
     depends_on('cmake')
-
-    def install(self, spec, prefix):
```

Comment:
By my count, around 120 Autotools packages and 30 CMake packages won't need an install method at all anymore :smiley: